addons/mail/tests/discuss/test_rtc.py

```diff
@@ -97,6 +97,7 @@ def test_10_start_call_in_chat_should_invite_all_members_to_call(self):
         with self.assertBus(
             [
                 (self.cr.dbname, 'discuss.channel', channel.id),  # update new session
+                (self.cr.dbname, 'discuss.channel', channel.id),  # channel_seen after posting join message
```

Comment:
We like to have the explicit payload for both, rather than deducing. The main issue is that we send 2 separate notifications rather that one with combined payload I think.
But I guess it's fine to keep this diff, and improve it later.